libcxx/include/__memory_resource/polymorphic_allocator.h

```diff
@@ -174,6 +174,17 @@ class _LIBCPP_AVAILABILITY_PMR _LIBCPP_TEMPLATE_VIS polymorphic_allocator {
 
   _LIBCPP_HIDE_FROM_ABI memory_resource* resource() const noexcept { return __res_; }
 
+  friend bool operator==(const polymorphic_allocator& __lhs, const polymorphic_allocator& __rhs) noexcept {
+    return *__lhs.resource() == *__rhs.resource();
+  }
+
+#  if _LIBCPP_STD_VER == 17
```

Comment:
```suggestion
#  if _LIBCPP_STD_VER <= 17
```
I find the current version quite confusing at first glance, since we don't do that usually.